Finmer.Editor/FormMain.cs

```diff
@@ -282,6 +282,23 @@ private bool SaveAs()
             if (dlgSave.ShowDialog() != DialogResult.OK)
                 return false;
 
+            else 
+            {
+                // Pull the directory we're saving in, see if there's any other finmer projects already there.
+                string save_path = Path.GetDirectoryName(dlgSave.FileName);
+                string[] file_entries = Directory.GetFiles(save_path, "*.fnproj");
+                if (file_entries.Length >= 1)
```

Comment:
If the user opens a project, then clicks Save As, and chooses to overwrite the exact same project file, this yields a false-positive.

I think we should examine the contents of `file_entries` and alert only if there is a file _other than_ the file we're going to save now.

Reply:
Thought I had that working properly last night, but testing again it did not.

I only added another short check to the if checking length, as the length >= 1 keeps it from firing when there isn't a .fnproj in the directory to begin with.